apps/server/src/routes/agent/mcp.ts

```
@@ -0,0 +1,475 @@
+export class ZeroMCP extends McpAgent<typeof env, {}, { userId: string }> {
+  server = new McpServer({
+    name: 'zero-mcp',
+    version: '1.0.0',
+    description: 'Zero MCP',
+  });
+
+  activeConnectionId: string | undefined;
+
+  constructor(ctx: DurableObjectState, env: Env) {
+    super(ctx, env);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)